app/views/glossaries/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Glossary</h1>
+<h2>Edit entry</h2>
+
+
+<%= form_for @glossary do |f| %>
+  <div>
+    <%= f.label :term %><br />
+    <%= f.text_field :term, autofocus: true, autocomplete: "term" %>
+  </div>
+
+  <div>
+    <%= f.label :definition %><br />
+    <%= f.text_field :definition, autofocus: true, autocomplete: "definition" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Update" %>
+  </div>
+
+  <%= link_to 'Cancel', glossaries_path %>
+
+<% end %>
```

Comment:
Surprised that rubocop didn't flag the lack of a newline at the end of the file here.